packages/types/src/index.ts

```diff
@@ -44,22 +44,28 @@ export interface ShowEvent {
 
 export interface ConnectEvent {
   is_atlas?: boolean;
+  resolved_hostname?: string;
   is_localhost?: boolean;
-  is_do?: boolean;
+  is_do_url?: boolean;
   server_version?: string;
+  mongosh_version: string;
```

Comment:
Similarly here – this is intentional? We always add this in `setup-logger-and-telemetry.ts` anyway

Reply:
Removed together with requiring package.json.